src/Slide.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
-import ReviewContainer from './ReviewContainer'
-import ReviewData from './ReviewData.js'
+import ReviewContainer from './SlideContainer'
+import ReviewData from './SlideData.js'

```

Comment:
As you have changed file names , as a good practice you could also change these names of module object(here ReviewContainer and ReviewData) so that person viewing the code would not get confused.